convert-stage-4.sh

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+
+show_help() {
+cat << EOF
+
+Mender executables, service and configuration files installer.
+
+Usage: $0 [options]
+
+    Options: [-i|--image | -m|--mender | -a|--artifact | -d|--device-type | -g|--go | -s|--server]
+
+        --image       - .sdimg generated with mender-conversion-tool
+        --mender      - mender client binary file
+        --artifact    - artifact info
+        --device-type - target device type identification
+        --server      - Mender server IP address
+
+    Examples:
+
+        ./mender-install-files.sh -i mender_beaglebone.sdimg -m mender
+                -a mender-image-1.4.0 -d beaglebone -s 192.168.10.2
+
+EOF
+}
+
+tool_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+output_dir=${tool_dir}/output
+GO_IMPORT="github.com/mendersoftware/mender"
+support_dir=$output_dir/support
+device_type=
+artifact=
+server_ip=
+
+declare -a sdimgmappings
+
+create_client_files() {
+  cat <<- EOF > $output_dir/mender.service
+	[Unit]
+	Description=Mender OTA update service
+	After=systemd-resolved.service
+
+	[Service]
+	Type=idle
+	User=root
+	Group=root
+	ExecStartPre=/bin/mkdir -p -m 0700 /data/mender
+	ExecStartPre=/bin/ln -sf /etc/mender/tenant.conf /var/lib/mender/authtentoken
```

Comment:
Above is not needed and was replaced with the "TenantToken" option in mender.conf

Reply:
Will be done in a separate commit.